tests/feature_extraction_test.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+
+# This source code is licensed under the license found in the
+# LICENSE file in the root directory of this source tree.
+
+import unittest
+from translation.feature_extraction_module import FeatureExtractionConfig, FeatureExtractionModule, LauncherConfig
```

Comment:
I think that this `tests` directory and the `translation` directory should go into the `src/ssvp_slt` so that you can import them normally e.g. `from ssvp_slt.translation.feature_extraction_module`